File: uri/1 - Basics/1020/Main.c
#include <stdio.h>

int main(){
    int a;
    scanf("%d", &a);
    printf("%d ano(s)\n", a/365);
    printf("%d mes(es)\n", (a%365)/30);
    printf("%d dia(s)\n", (a%365)%30);
    return 0;
}

File: uri/1 - Basics/1020/Main.go
package main

import "fmt"

func main() {
	var a int
	fmt.Scanf("%d", &a)
	fmt.Printf("%d ano(s)\n", a/365)
	fmt.Printf("%d mes(es)\n", (a%365)/30)
	fmt.Printf("%d dia(s)\n", (a%365)%30)

}
